app/containers/AudioPlayer/styles.ts

```diff
@@ -0,0 +1,69 @@
+import { StyleSheet } from 'react-native';
+
+import sharedStyles from '../../views/Styles';
+
+const styles = StyleSheet.create({
+	audioContainer: {
+		flex: 1,
+		flexDirection: 'row',
+		alignItems: 'center',
+		height: 56,
+		borderWidth: 1,
+		borderRadius: 4,
+		marginBottom: 6
```

Comment:
Not multiple of 4

Reply:
https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5160/files#diff-1b3b3e8ec8b635efad40ed7cc2db80c2ed04b26285046232b525cb3614a6d88cL74

I only use the same marginBottom as before

Reply:
You have a new design on Figma. We have new standards.